pbft/pbft.cpp

```diff
@@ -387,8 +391,15 @@ pbft::handle_request(const bzn_envelope& request_env, const std::shared_ptr<sess
 void
 pbft::forward_request_to_primary(const bzn_envelope& request_env)
 {
+    auto primary = this->get_current_primary();
+    if (!primary.has_value())
+    {
+        LOG(error) << "Would forward request to primary, but we don't know the primary so we can't";
+        return;
+    }
+
     this->node
-        ->send_signed_message(bzn::make_endpoint(this->get_primary()), std::make_shared<bzn_envelope>(request_env));
+        ->send_signed_message(bzn::make_endpoint(primary.value()), std::make_shared<bzn_envelope>(request_env));
```

Comment:
Not that we "support" Mac, but .value() seems to cause a problem with Mac. Possibly use *primary instead.